extensions/wikidata/src/org/openrefine/wikidata/commands/AuthorizeCommand.java

```diff
@@ -0,0 +1,39 @@
+package org.openrefine.wikidata.commands;
+
+import com.google.refine.commands.Command;
+import org.openrefine.wikidata.editing.ConnectionManager;
+
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.io.Writer;
+
+/**
+ * Command for redirecting the user to the authorization page.
+ */
+public class AuthorizeCommand extends Command {
+
+    @Override
+    public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException, ServletException {
+        ConnectionManager manager = ConnectionManager.getInstance();
+        try {
+            if (manager.supportOAuth()) {
+                String remember = request.getParameter("remember-credentials");
+                manager.setRememberCredentials("on".equals(remember));
+                String authorizationUrl = manager.getAuthorizationUrl();
+                response.sendRedirect(authorizationUrl);
+            } else {
+                throw new IllegalStateException("You must configure Wikidata OAuth client id/secret in refine.ini " +
```

Comment:
This probably should be an error code or something which gets interpretted on the front end so that we can take advantage of the i18n support there.